libs/client/ui/pages/src/lib/notifications/notifications.tsx

```diff
@@ -0,0 +1,153 @@
+  userStore?: UserStore;
+}
+
+export const Notifications: React.FC<NotificationsProps> = inject(
+  notificationsStore.storeKey,
+  userStore.storeKey
+)(
+  observer(({ notificationsStore, userStore }) => {
+    const [presentDefaultErrorToast] = useDefaultErrorToast();
+    const [isGuest] = useIsGuest();
+
+    const markNotificationAsRead = (notificationId: Notification['id']) => {
+      notificationsStore
+        ?.markNotificationAsRead({ notificationId })
+        .then(() => notificationsStore?.fetchNotifications())
+        .catch(() => {
+          presentDefaultErrorToast();
+        });
+    };
+
+    const deleteNotification = (notificationId: Notification['id']) => {
+      notificationsStore
+        ?.deleteNotification({ notificationId })
+        .then(() => notificationsStore?.fetchNotifications())
+        .catch(() => {
+          presentDefaultErrorToast();
+        });
+    };
+
+    const handleRefresh = (e: any) => {
+      if (isGuest) return setTimeout(() => e.target.complete(), 500);
+
+      return notificationsStore
+        ?.fetchNotifications()
+        .then(() => e.target.complete())
+        .catch(() => {
+          presentDefaultErrorToast();
+        });
+    };
+
+    return (
+      <IonPage className="notifications">
+        <IonHeader mode="ios">
+          <IonToolbar>
+            <IonButtons>
+              <IonBackButton
+                color="primary"
+                defaultHref={`${tabRoutes.start.route}`}
+                text="Zurück"
+              />
+            </IonButtons>
+            <IonTitle>Benachrichtigungen</IonTitle>
+          </IonToolbar>
+        </IonHeader>
+        <IonContent className="notifications__content">
+          <IonRefresher slot="fixed" onIonRefresh={handleRefresh}>
+            <IonRefresherContent
+              refreshingSpinner={`crescent`}
+            ></IonRefresherContent>
+          </IonRefresher>
+          {notificationsStore?.notifications &&
+          notificationsStore.notifications?.length > 0 ? (
+            notificationsStore?.notifications.map((notification, i) => {
+              return (
+                <IonItemSliding
+                  key={`${JSON.stringify(toJS(notification))}-${i}`}
+                >
+                  {!notification.read ? (
+                    <div className="unread-indicator"></div>
+                  ) : undefined}
+                  <IonItem
+                    detail
+                    routerLink={
+                      notification.type === NotificationType.FRIEND_REQUEST
+                        ? `${tabRoutes.profile.route}/friends`
+                        : undefined
+                    }
+                    mode="ios"
+                    className={`${!notification.read ? 'unread' : ''}`}
+                  >
+                    <IonLabel>
+                      <h2>{notification.title}</h2>
+                      <p>{notification.description}</p>
```

Comment:
Eventuell die Ionic elems verwenda

Reply:
Hot mi o gwundert aba isch sogar so in da ionic documentation dinna